lib/widgets/login.dart

```diff
@@ -343,6 +343,9 @@ class _LoginPageState extends State<LoginPage> {
       // Could set [_inProgress]… but we'd need to unset it if the web-auth
       // attempt is aborted (by the user closing the browser, for example),
       // and I don't think we can reliably know when that happens.
+
+      // Not using [PlatformActions.launchUrl] because web auth needs special
+      // error handling.
```

Comment:
Perhaps as a part of #462, we can get rid of the block of iOS specific error handling code and reuse `PlatofmActions.launchUrl`, but I'm hesitant because of the question mark from `// TODO(#462) remove this?`.

Reply:
Yeah; that sounds like it would need investigating when we get to #462.